integration/restart_linux_test.go

```diff
@@ -76,3 +78,42 @@ func TestContainerdRestartSandboxRecover(t *testing.T) {
 	}
 	assert.True(t, foundUnkownSb)
 }
+
+func TestRestartContainerdTime(t *testing.T) {
+	workDir := t.TempDir()
+
+	currentReleaseCtrdDefaultConfig(t, workDir)
+
+	ctrd := newCtrdProc(t, *containerdBin, workDir, []string{})
+
+	logPath := ctrd.logPath()
+	t.Cleanup(func() {
+		if t.Failed() {
+			dumpFileContent(t, logPath)
+		}
+	})
+	require.NoError(t, ctrd.isReady())
+
+	podCtxs := []*podTCtx{}
+	defer func() {
+		for _, p := range podCtxs {
+			p.stop(true)
+		}
```

Comment:
sorry for that my last comment. we do need the cleanup.

Maybe we should call it here.

```
defer func() {
		for _, p := range podCtxs {
			p.stop(true)
		}
         assert.NoError(t, ctrd.kill(syscall.SIGTERM))
	assert.NoError(t, ctrd.wait(5*time.Minute))
}()
```